util/log/clog.go

```diff
@@ -630,6 +630,7 @@ type loggingT struct {
 	// does not let us avoid the =true, and that shorthand is necessary for
 	// compatibility. TODO: does this matter enough to fix? Seems unlikely.
 	toStderr        bool          // The -logtostderr flag.
+	json            bool          // The -logjson flag.
 	alsoToStderr    bool          // The -alsologtostderr flag.
```

Comment:
How about changing flags to:

  -logtostderr=(proto|json|plain|none)
  -logtofile=(proto|json|plain|none)
  -logdir=<logging directory>
By the way, update the -alsologtostderr in the comment at line 634 to -alsotostderr.